transforms/goog_module_to_es6_module.js

```diff
@@ -37,6 +38,15 @@ module.exports = (info, api, options) => {
       }
       currentModuleSymbol = path.value.expression.arguments[0].value;
       path.replace();
+
+      // add @module annotation
+      if (comments) {
+        for (const comment of comments) {
+          if (comment.value.indexOf('@module') >= 0) {
+            hasModule = true;
```

Comment:
Use addModule and check the module name matches the existing value. If not we need to throw.